makePoplogCommander.sh

```diff
@@ -371,6 +374,16 @@ fi
 echo "$CODE1"
 echo 
 
+################################################################################
+# Note that poplocal needs special handling. 
```

Comment:
I don't think this is a very helpful comment. It doesn't explain why `poplocal` needs special handling. I think it should at least link to https://github.com/GetPoplog/Seed/wiki/A-new-default-for-$poplocal for context.

Reply:
Uh, yeah. You got me. I will fix!

Reply:
Fixed